vllm/v1/structured_output/backend_outlines.py

```diff
@@ -0,0 +1,310 @@
+# SPDX-License-Identifier: Apache-2.0
+from __future__ import annotations
+
+import ast
+import json
+import re
+import sys
+from dataclasses import dataclass, field
+from typing import TYPE_CHECKING
+
+import torch
+
+from vllm.config import VllmConfig
+from vllm.model_executor.guided_decoding.outlines_logits_processors import (
```

Comment:
This is out of scope for this PR, but we will probably move this to the utils file under V1 once V0 is removed in 0.10.x